testing/cli/test_sct_deepseg.py

```diff
@@ -174,4 +175,4 @@ def test_segment_nifti_multiclass(fname_image, fnames_seg_manual, fname_out, suf
             im_seg = Image(fname_out)
             im_seg_manual = Image(fname_seg_manual)
             dice_segmentation = compute_dice(im_seg, im_seg_manual, mode='3d', zboundaries=False)
-            assert dice_segmentation > 0.95
+            assert dice_segmentation > 0.85  # Temporary until we improve "fake_lesion" ground truth
```

Comment:
Todo: 

- [ ] Replace the ground truth file inside of `sct_testing_data`
- [ ] Create a new release
- [ ] Update the URL inside this PR
- [ ] Revert the change to the target dice score

Reply:
Done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4583/commits/9ed7a4d09c2861cb4268b5fa8029350f4d270cfd.